### ISSUE TITLE
Logo should have alt and aria-label text

### ISSUE THREAD

**Issue by sirkitree** [Lullabot/lb-ebook-poc#41]
From DubBot
![image](https://github.com/user-attachments/assets/b449106b-8d5d-4964-be14-4ff7445102b4)
